nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/AccessPolicyResource.java

```diff
@@ -102,14 +102,18 @@ public AccessPolicyEntity populateRemainingAccessPolicyEntityContent(AccessPolic
     @Consumes(MediaType.WILDCARD)
     @Produces(MediaType.APPLICATION_JSON)
     @Path("{action}/{resource: .+}")
-    // TODO - @PreAuthorize("hasAnyRole('ROLE_MONITOR', 'ROLE_DFM', 'ROLE_ADMIN')")
     @ApiOperation(
-            value = "Gets an access policy",
+            value = "Gets an access policy for the specified action and resource",
+            notes = "Will return the effective policy if no component specific policy exists for the specified action and resource. "
+                    + "Must have Read permissions to the policy with the desired action and resource. Permissions for the policy that is "
+                    + "returned will be indicated in the response. This means the client could be authorized to get the policy for a "
+                    + "given component but the effective policy may be inherited from an ancestor Process Group. If the client does not "
+                    + "have permissions to that policy, the response will not include the policy and the permissions in the response "
+                    + "will be marked accordingly. If the client does not have permissions to the policy of the desired action and resource "
+                    + "a 403 response will be returned.",
             response = AccessPolicyEntity.class,
             authorizations = {
-                    @Authorization(value = "Read Only", type = "ROLE_MONITOR"),
-                    @Authorization(value = "Data Flow Manager", type = "ROLE_DFM"),
-                    @Authorization(value = "Administrator", type = "ROLE_ADMIN")
+                    @Authorization(value = "Read - /policies/{action}/{resource}", type = "")
```

Comment:
I think the authorization documentation for all of the methods in this resource should be for "/policies/{resource}" because if you make a policy for a policy, the resource doesn't include the action. For example, for the root group if I make a policy for viewing it's policies, I get:

resource="/policies/process-groups/50aca6b9-0156-1000-ad28-19135eccdef7" action="R"